configuration/restdocs.md

```diff
@@ -7,14 +7,38 @@ title: openHAB REST API
 
 # openHAB REST API
 
-The REST API of openHAB serves different purposes.
-It can be used to integrate openHAB with other systems as it allows read access to items and item states as well as status updates or the sending of commands for items.
-Furthermore, it gives access to the sitemaps, so that it is the interface to be used by remote user interfaces (e.g. fat clients or javascript based web clients).
-The REST API also supports server-push, so you can subscribe yourself on change notification for certain resources.
+Through the openHAB [REST API](https://en.wikipedia.org/wiki/REST_API) most aspects of the openHAB system can be accessed by other programs.  
+This includes for example the access to all data related to Items, Things and Bindings as well as the capabilities to invoke  actions that change the state of Items. Interactions with the REST API are based on a simple http protocoll.
```

Comment:
So - just practicing - change protocoll to protocol

Reply:
@LightIsLife it looks to me as if there are still a number of my comments which have not been addressed in your latest commit.  I hope I am looking at things correctly.  An example is that the word 'protocol' is still spelled 'protocoll' in the version I see when I use the 'view' button to see the entire document without all the diff coding.